contracts/Octobay.sol

```diff
@@ -405,17 +412,22 @@ contract Octobay is Ownable, ChainlinkClient, BaseRelayRecipient {
         recordChainlinkFulfillment(_requestId)
     {
         require(issueWithdrawRequests[_requestId].payoutAddress != address(0) );
+        address payoutAddr = issueWithdrawRequests[_requestId].payoutAddress;
         uint256 payoutAmt = issueDepositsAmountByIssueId[issueWithdrawRequests[_requestId].issueId];
         issueDepositsAmountByIssueId[issueWithdrawRequests[_requestId].issueId] = 0;
         issueStatusByIssueId[issueWithdrawRequests[_requestId].issueId] = IssueStatus.CLAIMED;
-        payable(issueWithdrawRequests[_requestId].payoutAddress).transfer(payoutAmt);
+        emit WithdrawIssueDepositEvent(issueWithdrawRequests[_requestId].issueId, payoutAddr, payoutAmt);
+        delete issueWithdrawRequests[_requestId];
         // delete issueDeposits[_depositId]; ??? loop through issueDepositIdsByIssueId ???
+        // awardGovernanceTokens(payoutAddr, payoutAmt, tokenAddr); // How do we go from issueId => tokenAddr ?
```

Comment:
Good question. I guess we need to store the repoId/orgId together with the issueId. Question is how... just passing it to the function will probably not be enough. Do we need to make sure that issueId and repoId/orgId actually belong to each other? hmm....